adoc/retail_chap_install.adoc

```diff
@@ -225,11 +227,11 @@ Here is a channel overview:
 +
 *TODO*: add more info about these 3 actions if wanted; it's in Advanced Topics, Proxy QS.
 +
-*TODO*: do we have a suma_retail_proxy pattern?
+*TODO*: do we have a suma_retail_proxy pattern? According to Jiri: yes.
```

Comment:
May be a misunderstanding: We have the suma_proxy pattern (not retail specific) and we have a retail pattern for the SUMA Server (but not proxy)